modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -618,65 +621,92 @@ The output represents the transformed XML elements into JSON.
+ Valid values are `true` or `false`.
 |===
 
+[[writer_properties]]
 === Writer Properties
 
-The XML format accepts properties that provide instructions for writing output data.
+This format accepts properties that provide instructions for writing output data.
 
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-| `bufferSize` | `Number` | `8192` | Size of the buffer writer.
-| `encoding` | `String` | `null` |Encoding for the writer to use.
-| `deferred` | `Boolean` | `false` |When set to `true`, DataWeave generates the output as a data stream, and the script's execution is deferred until it is consumed.
-  Valid values are `true` or `false`.
-| `escapeCR` | `Boolean` | `false` | Whether to escape a carriage return (CR).
-Valid values are `true` or `false`. Available since Mule 4.2.1.
-| `indent` | `Boolean` | `true` |Indicates whether to indent the code for better readability or to compress it into a single line.
-Valid values are `true` or `false`.
-| `inlineCloseOn` | `String` | `empty` |Indicates when the writer uses inline close tag or use open close tags.
-Valid values are `empty` or `none`.
-| `onInvalidChar` | `String` | `null` | Valid values are `base64`, `ignore`, or `none`.
-| `skipNullOn` | `String` | `null` | Skips `null` values in the specified data
-structure. By default, the writer does not skip them. Valid values are `elements`, `attributes`, or `everywhere`.
-| `writeDeclaration` | `Boolean` | `true` |Indicates whether to write the XML
-header declaration. Valid values are `true` or `false`.
-| `writeNilOnNull` | `Boolean` | `false` | Indicates whether to write a nil attribute when the value is `null`. Valid values are `true` or `false`.
+|Parameter |Type |Default|Description
+|`bufferSize`  |`Number`|`8192`|Size of the buffer writer.
+|`defaultNamespace`  |`String`|`null`|Specifies the default namespaces of the output XML.
+|`deferred`  |`Boolean`|`false`|Generates the output as a data stream when set to `true`, and defers the script's execution until consumed.
+ Valid values are `true` or `false`.
+|`doubleQuoteInDeclaration`  |`Boolean`|`false`|Escapes double quotes in the XML declaration when set to `true`.
+ Valid values are `true` or `false`.
+|`encoding`  |`String`|`null`|The character set to use for the output, such as UTF-8.
+|`escapeCR`  |`Boolean`|`false`|Escapes `CR` characters when set to `true`.
+ Valid values are `true` or `false`.
+|`escapeGT`  |`Boolean`|`false`|Escapes '>' characters when set to `true`.
+ Valid values are `true` or `false`.
+|`indent`  |`Boolean`|`true`|Indicates whether to indent the JSON code for better readability or to compress the JSON into a single line. Valid values are `true` or `false`.
```

Comment:
TODO: indent (for XML) COPYEDIT This is talking about JSON. Need to make more generic or fix somehow. Also remove extra "Valid values..."

Reply:
"Indicates whether to indent the code for better readability or to compress it into a single line."